services/src/main/java/org/exoplatform/news/NewsServiceImpl.java

```diff
@@ -391,6 +398,29 @@ public News updateNews(News news) throws Exception {
           indexingService.reindex(NewsIndexingServiceConnector.TYPE, String.valueOf(news.getId()));
         } else if ("draft".equals(news.getPublicationState())) {
           publicationService.changeState(newsNode, "draft", new HashMap<>());
+          Identity identity = identityManager.getOrCreateIdentity(OrganizationIdentityProvider.NAME, getCurrentUserId());
+          String currentUserId = identity.getId();
+          if (!newsNode.isNodeType("mix:newsModifiers")) {
+            newsNode.addMixin("mix:newsModifiers");
+          }
+          Value[] modifiersIdsProperty;
+          List<String> modifiersIds = new ArrayList<>();
+          boolean alreadyExist = false;
+          if (newsNode.hasProperty("exo:newsModifiersIds")) {
```

Comment:
I think that this test is useless since we have the mixin added and the property existing